### PR TITLE
fix regulatory calcualtions and add example on docs

### PR DESCRIPTION
**Pull request by rschwant**
1) Fix MDA8 ozone calculations
2) Add two days of output to ftp site for testing and ReadTheDocs example
3) Add ReadTheDocs example for regulatory options
4) Update a couple items to get rid of warnings (i.e. add numeric_only=True in two areas of driver)
5) Add description for how read in code with the conda environment, so we are ready for this for next week
6) Fix regulatory example in examples folder to exclude spatial overlay plot for consistency with ReadTheDocs
7) Update docs in supported diagnostics to include 2 plots that do not work for regulatory options
8) Updated version to 0.1 in prep for conda release

### CODE REVIEW

**Comment by blychs** [comment]
This actually looks great, Becky!

**Review by blychs — APPROVED** [review]
This looks great! We have to fix the Pandas issues, but wouldn't change it until we are sure.

File: docs/getting_started/installation.rst [hunk 1]
```diff
@@ -30,20 +30,34 @@ General instructions
...
+installing it is relatively simple. There are two methods available.
+
+Option 1) Using Conda
+^^^^^^^^^^^^^^^^^^^^^
+We have recently created a conda-forge release of MELODIES MONET to make installation very simple 
+with just 3 lines of code below::
+
+    $ conda create --name melodies-monet python=3.9
+    $ conda activate melodies-monet
+    $ conda install -y -c conda-forge melodies-monet wrf-python jupyterlab
```
Comment (zmoon) [inline docs/getting_started/installation.rst:42]:
```suggestion
with just 1 line of code below::

    $ conda create --name melodies-monet -y -c conda-forge python=3.9 melodies-monet wrf-python jupyterlab
```

File: docs/index.rst [hunk 2]
```diff
@@ -80,6 +80,7 @@ MONETIO please refer to:
    examples/airnow_wrfchem
    examples/airnow_camchem_se
    examples/camchem
+   examples/airnow_wrfchem_reg
```
Comment (zmoon) [inline docs/index.rst:83]: Maybe put it after airnow_wrfchem instead?

Comment (rschwant) [reply docs/index.rst:83]: Yes, we could move it there too. I'll move it when I fix Beiming's CSI plot label too. Will do this now.

**Review by blychs — APPROVED** [review]
Great, thanks @rschwant!

**Pull request merged**